### PR TITLE
[INLONG-2301] add standalone deployment  shell

### LINKED ISSUE #2301: [Feature]  Support Standalone deployment for InLong
### Description

#1435 support deployment using docker-compose, we need a standalone deployment method without for developer or test.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by crancchen**
add standalone deployment  shell

### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes #2301

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: initInlongConfig.sh [hunk 1]
```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+#初始化inlong 各组件的配置文件
```
Comment (luchunliang) [inline initInlongConfig.sh:2]: Please change to English.

File: initInlongConfig.sh [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+source $basePath/inlongConfig.properties
+
+echo  "1.替换agent配置参数"
+cd $basePath/inlong-agent/conf
+sed -i 's/agent.http.enable=.*/'''agent.http.enable=${agent_http_enable}'''/g' agent.properties
+sed -i 's/agent.local.ip=.*/'''agent.local.ip=${agent_local_ip}'''/g' agent.properties  
+sed -i 's/agent.manager.vip.http.host=.*/'''agent.manager.vip.http.host=${manager_server_hostname}'''/g' agent.properties 
+sed -i 's/agent.manager.vip.http.port=.*/'''agent.manager.vip.http.port=${manager_server_port}'''/g' agent.properties 
+
+echo "2.替换dataproxy配置参数"
```
Comment (luchunliang) [inline initInlongConfig.sh:25]: ditto

File: stopInlongCluster.sh [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+else
+   tubemq_master_pid=$(echo $tubemq_master_thread | tr -cd "[0-9]")
+   kill -9 $tubemq_master_pid
+ echo "tubemq_master stopped "
+fi
+
+echo "7.stop CliFrontend... "
+cli_frontend_thread=$($JAVA_HOME/bin/jps |grep CliFrontend)
+if [ ! -n "$cli_frontend_thread" ]; then 
+   echo "系统没启动cli_frontend"
```
Comment (luchunliang) [inline stopInlongCluster.sh:66]: ditto

File: bin/inlong-daemon.sh [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+    cli_frontend_thread=$($JAVA_HOME/bin/jps |grep CliFrontend)
+    if [ ! -n "$cli_frontend_thread" ]; then 
+       echo "The system did not running cli_frontend" 
+    else
+       cli_frontend_pid=$(echo $cli_frontend_thread | tr -cd "[0-9]")
+       kill -9 $cli_frontend_pid
+       echo "cli_frontend stopped "
+    fi
+	
+	echo "8.stop sort flink job"
```
Comment (kipshi) [inline bin/inlong-daemon.sh]: format code

**Review by luchunliang — APPROVED** [review]

**Comment by crancchen** [comment]
add  standalone  deployment shell

**Review by dockerzhang — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

**Pull request merged**